pkg/storage/streams/store.go

```diff
@@ -59,16 +62,19 @@ type Store interface {
 
 // streamStore is a store for streams
 type streamStore struct {
-	segments    segments.Store
-	segmentSize int64
+	segments           segments.Store
+	segmentSize        int64
+	key                string
+	encryptedBlockSize int
```

Comment:
This is the "block size used for encryption", not "a block size that is encrypted itself". So let's avoid the confusion and name it as "encryptionBlockSize".